src/sentry/replays/consumers/recording.py

```diff
@@ -79,28 +81,57 @@ def create_with_partitions(
 # Processing Task
 
 
-def process_message_with_options(
+def _get_replay_profiling_project_key():
+    """Get or create a project key specifically for replay consumer profiling data."""
+    try:
+        internal_project = Project.objects.get(slug="sentry-replay-consumer-profiling")
+        profiling_key, _ = ProjectKey.objects.get_or_create(
+            use_case=UseCase.PROFILING.value,
+            project=internal_project,
+            defaults={"label": "Replay Consumer Profiling"},
+        )
+        return profiling_key
+    except Project.DoesNotExist:
+        logger.warning("Replay profiling project is not configured")
+        return None
```

Comment:
can we inject this value instead of fetching it from the database? i suspect that fetching it will add a bunch of overhead to the task we're measuring.